blazer/app/views/blazer/queries/show.html.erb

```diff
@@ -19,6 +19,10 @@
       <div class="col-sm-3 text-right">
         <%= link_to "Edit", edit_query_path(@query, params: variable_params(@query)), class: "btn btn-default", disabled: !@query.editable?(blazer_user) %>
         <%= link_to "Fork", new_query_path(params: variable_params(@query).merge(fork_query_id: @query.id, data_source: @query.data_source, name: @query.name)), class: "btn btn-info" %>
+
+        <% if !@error && @success %>
+          <%= button_to "Download", run_queries_path(format: "csv"), params: run_data, class: "btn btn-primary", data: {confirm: "Please confirm there is no protected data before downloading"} %>
```

Comment:
Do we have a resource that goes over what is considered protected data in our system that we could link to in the confirmation message?
It should be fairly straightforward what is and is not protected if you think about it, but it wouldn't hurt to "help" the user think about it if that makes sense.

Reply:
I think @Pensai was thinking of that slide:
https://docs.google.com/presentation/d/1A_0CAcORZI0vB_if3L53IzEJwr8cr0N8aJ5UmtWXwHc/edit#slide=id.g110613c5ef7_0_814

We don't have a clear identification of which columns in the database are sensitive. Hypershield would manage that automatically. It would be good to update our database comments field on column as well so it propagates to [our documentation over there](https://cds-snc.github.io/notification-documentation-database/tables/notifications.html) and we could link to this if people are uncertain what's sensitive or not.